projects/asurcosg.yaml

```diff
@@ -4,8 +4,8 @@ FieldOfScience: Computer and Information Science and Engineering
 ID: '426'
 Name: asurcosg
 Organization: Arizona State University
-PIName: Jay Etchings
+PIName: Johnathan Lee
 Sponsor:
-  CampusGrid:
-    ID: '14'
-    Name: OSG Connect
+  VirtualOrganization:
+    ID: 30
+    Name: OSG
```

Comment:
Let's not make any changes to the project for now

Reply:
OK